modules/ROOT/pages/understanding-enterprise-integration-patterns-using-mule.adoc

```diff
@@ -0,0 +1,92 @@
+= Understanding Enterprise Integration Patterns Using Mule
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Enterprise Integration Patterns are accepted solutions to recurring problems within a given context. The patterns provide a framework for designing, building messaging and integration systems, as well as a common language for teams to use when architecting solutions.
+
+Mule supports most of these patterns shown in the Enterprise Integration Patterns book written by Gregor Hohpe and Bobby Woolf.
+
+By implementing these patterns, Mule reduces the effort required when building integrations. You can use these well-known patters to design your solution and then simply configure and use these same patters in Mule.
```

Comment:
"...Mule reduces the effort required when building integrations. You can use these well-known patters to design your solution and then simply configure and use these same patters in Mule."
   >  "...Mule reduces your work by enabling you to design, build, and use your own integrations based on these same patterns."

Reply:
I now notice that there is just one subtopic to this main topic, which leads me to wonder if we should combine the two into one Level-1 topic with tables.

Consider a title like "How Enterprise Integration Patterns Map to Mule Objects," followed by the current text explaining how Mule support of these patterns makes user's lives easier and maybe ending with something like this:

     "The enterprise integration patterns supported by Mule runtime engine are these:"

Then present the tables.